community/kernel/src/main/java/org/neo4j/kernel/impl/api/cursor/TxSingleRelationshipCursor.java

```diff
@@ -19,23 +19,27 @@
  */
 package org.neo4j.kernel.impl.api.cursor;
 
+import org.neo4j.cursor.Cursor;
 import org.neo4j.function.Consumer;
-import org.neo4j.kernel.api.cursor.RelationshipCursor;
+import org.neo4j.kernel.api.cursor.RelationshipItem;
 import org.neo4j.kernel.api.txstate.TransactionState;
 
 /**
- * Overlays transaction state on a {@link RelationshipCursor}.
+ * Overlays transaction state on a {@link RelationshipItem} item.
  */
 public class TxSingleRelationshipCursor
         extends TxAbstractRelationshipCursor
 {
+    private long id;
```

Comment:
`TxAbstractRelationshipCursor` already have `id` field do we need it here as well?